### PR TITLE
Web driver extension Yauheni

### PR DESCRIPTION
**Pull request by evgkas**

### CODE REVIEW

**Review by sirojbekdev — APPROVED** [review]

**Review by khayitmuradov — APPROVED** [review]
lgtm!

File: UI/Utils/WebDriverExtension.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+namespace UI.Utils
+{
+    public static class WebDriverExtension
+    {
+        private static void WaitElementIsVisible(By locator, int timeSeconds)
+        {
+            WebDriverWait wait = new WebDriverWait(Browser.GetDriver(), TimeSpan.FromSeconds(timeSeconds));
+            wait.Until(driver => driver.FindElement(locator).Displayed);
+        }
+        public static void ClickOnElement(By locator, int waitSeconds)
```
Comment (NikitaGavruk) [inline UI/Utils/WebDriverExtension.cs]: Let's call all timeout in one pattern e.g: timeSeconds

File: UI/Utils/WebDriverExtension.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            Browser.GetDriver().FindElement(locator).Click();
+        }
+
+        public static void ClickOnElement(By locator)
+        {
+            WaitElementIsVisible(locator, 5);
+            Browser.GetDriver().FindElement(locator).Click();
+        }
+
+        public static void SendKeysToElement(By locator, string text)
```
Comment (NikitaGavruk) [inline UI/Utils/WebDriverExtension.cs]: Let's create all functions from that file with timeout parameter like on 10 line

**Review by NikitaGavruk — CHANGES_REQUESTED** [review]

File: UI/Utils/WebDriverExtension.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            WebDriverWait wait = new WebDriverWait(Browser.GetDriver(), TimeSpan.FromSeconds(timeSeconds));
+            wait.Until(driver => driver.FindElement(locator).Displayed);
+        }
+        public static void ClickOnElement(By locator, int timeSeconds)
+        {
+            WaitElementIsVisible(locator, timeSeconds);
+            Browser.GetDriver().FindElement(locator).Click();
+        }
+
+        public static void ClickOnElement(By locator)
```
Comment (NikitaGavruk) [inline UI/Utils/WebDriverExtension.cs]: Delete that duplicate of function above

File: UI/Utils/WebDriverExtension.cs [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            {
+                return false;
+            }
+            catch (StaleElementReferenceException)
+            {
+                return false;
+            }
+        }
+
+        public static string GetAttributeValueFromElement(By locator, int timeSeconds, string attribute)
```
Comment (NikitaGavruk) [inline UI/Utils/WebDriverExtension.cs]: Let's make timeout last parameter on functions

**Comment by evgkas** [comment]
I leave one method without timeout. There is also overload with timeout

**Review by NikitaGavruk — CHANGES_REQUESTED** [review]

**Comment by NikitaGavruk** [comment]
> I leave one method without timeout. There is also overload with timeout

We do not need it

**Pull request merged**